components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -23,6 +23,9 @@ use dom::bluetoothuuid::{BluetoothDescriptorUUID, BluetoothUUID};
 use ipc_channel::ipc::{self, IpcSender};
 use net_traits::bluetooth_thread::BluetoothMethodMsg;
 
+// Maximum length of an attribute value.
+const MAXIMUM_ATTRIBUTE_LENGTH: usize = 512;
```

Comment:
https://www.bluetooth.org/DocMan/handlers/DownloadDoc.ashx?doc_id=286439 (Vol. 3, page 2169) and you can find it in the specification steps too: https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattcharacteristic-writevalue (Step 4).